Land_Sector_Data_Analysis/Land_Sector_Dataset_Description.md

```diff
@@ -0,0 +1,290 @@
+
+The Climate directory contains the Koppen Geiger climate shift from 1901 to 2100. The files within this subdirectory have the follwing attributes:
+
+	OBJECTID	ID	GRIDCODE	Shape_Length	Shape_Area	geometry
+
+  some files within the subdirectory have the suffixes: 
+  >A1FI (fossil fuel intensive),
+
+  >A1 ( new and efficient technologies), 
+
+  >A2 (a very heterogenic world with focus on family values and local traditions),
+
+  >B1 (a world without materialism and launch of
+clean technologies), 
+
+  >B2 (a world with focus on local solutions for economic and ecological sustainability),  
+
+
+Within the climate directory the files:
+
+>  **IPCC_ClimateZoneMap.tif:** Contains the geotiff IPCC climate zone map.
+
+>  **IPCC_ClimateZoneMap_Vector.geojson:** contains the vectorised format fo the geotiff IPCC climate zone map. This file has the properties: 
+	CLASS_NAME	geometry
+
+>  **KoppenGeigerClimateShifts.zip**: This file contains the zips of the Koppen Geiger Climate shifts.
+
+
+# 4. Land Cover
+
+
+> The Land ccover directory has four principal subdirectories:
+
+
+### 4.1 Biomass Carbon: The Biomas Carbon is further subdivided into:
+
+#####4.1.1 Above Ground Live Biomass: 
+This directory conains a zip file( AboveGroundLiveBiomassTiles.zip) with dataset on all living biomass above the soil including stem, stump, branches, bark, seeds and foliage.
+
+##### 4.1.2 Avitabile Pan Tropical Biomass:
+This directory contains Avitabile's Above ground biomass geotiff and zip files
+
+##### 4.1.3 Geo Carbon Global Forest Biomass Maps: 
+Contains geo carbon geotiff and zip file maps(global forest biomass, above ground biomass, and forest above ground biomass)
+
+##### 4.1.4 Global Biomass
+This directory has 03 global biomass subdirectories containing tiles for above ground biomass, growing stock volums and originals. The file names in these sub directories directly imply the tiles of interest.
+
+
+### 4.2 Forest
+This directory contains subdirectories:
+
+##### 4.2.1 Managed Forest Concessions: 
+Within this directory there are files containing managed forest concessions of 10 countries including the global geojson file for managed forest concessions. These files have the following properties.
+  **id,	OBJECTID,	country,	name,	company,	group_comp,	group_coun,	legal_term,	status,	province,	source,	last_updat,	Shape_Leng,	AREA_GEO,	Shape_Length,	Shape_Area, and	geometry**
+
+ ##### 4.2.2 Pan Tropical Forest Strata
+  Contains geotiff files of Pan tropical forest strata for Africa(AFR), South America(SAM) and Southeast Asia(SEA) as well as their relative zip files.
+
+ ##### 4.2.3 Planted forests
+  This data is from The Spatial Database of Planted Trees(SDPT). It contains data on 43 countries. Each file has the follwing properties:
+  	**final_id,	iso,	country,	org_name,	common_name,	species,	species_simp,	plant_ag,	timber_ag,	ever_dec,	conifer_broad,	hard_soft,	size,	source,	year,	geometry**
+
+ ##### 4.2.4 Wood Fiber Concessions
```

Comment:
```suggestion
 ### 4.2.2 Pan Tropical Forest Strata
  Contains geotiff files of Pantropical forest strata for Africa(AFR), South America(SAM), and Southeast Asia(SEA) as well as their relative zip files.

 ### 4.2.3 Planted forests
  This data is from The Spatial Database of Planted Trees(SDPT). It contains data on 43 countries. Each file has the follwing properties:
  	**final_id,	iso,	country,	org_name,	common_name,	species,	species_simp,	plant_ag,	timber_ag,	ever_dec,	conifer_broad,	hard_soft,	size,	source,	year,	geometry**

 ### 4.2.4 Wood Fiber Concessions
```

Reply:
Thank you @chicken-biryani . I have adopted the changes, please let me know if there are more changes to be made. I used the Markdown Formatter, its such a great took. Thanks again.